app/code/Magento/Catalog/view/frontend/templates/product/view/gallery.phtml

```diff
@@ -12,6 +12,16 @@
 ?>
 
 <?php
+// If block argument "gallery_options" is not set in xml layout file, then fallback
+// to an instance of \Magento\Catalog\Block\Product\View\GalleryOptions class manually
+if ($block->getGalleryOptions() === null) {
+    $block->setData(
+        'gallery_options',
+        \Magento\Framework\App\ObjectManager::getInstance()
+            ->get(\Magento\Catalog\Block\Product\View\GalleryOptions::class)
```

Comment:
Can we obtain it from layout somehow or it's not present? Using `ObjectManager` in template is quite messy approach.